monitoring/README.md

```diff
@@ -92,47 +157,108 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
+| ------------- | ------------- |
+| Host-cluster  |     6443      |
+| WDS space     |     6443      |
+| ITS space     |     8443      |
+| WEC-cluster   |     9444      |
 
-Note: you can also monitor the APIserver of the hosting cluster using the following values: `NS: default` and `APIService: apiserver`
 
 b) API Server Priority & Fairness: input into the following text boxes.
 - `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, *its1-system*, etc.
 - `APIService`: name of the service for a KubeStellar space APIserver, e.g., *wds1*, *wds2*, *vcluster*, etc.
+- `Instance`: hostname and port number for the target KubeStellar space APIserver, e.g., *10.244.0.18:9444*
 
     <img src="images/apiserver-apf-monitoring.png" width="60%" height="80%" title="apiserver-apf-monitoring.png"> 
 
-Note: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space or the hosting cluster. 
+**Note**: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space, the hosting or WEC clusters. 
 
 
 c) KubeStellar Controllers Monitoring: input into the following text box and drop-down menu. 
 
-- `SpaceNS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
+- `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
 
-- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller* and *status-addon-controller*. 
+- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller*, *status-addon-controller* and *status-agent-controller*. 
+
+- `Instance`: hostname and port number for the target KubeStellar space controller, e.g., *10.244.0.71:8090*
 
     <img src="images/ks-controllers-monitoring.png" width="60%" height="80%" title="ks-controllers-monitoring">
 
-To monitor the KubeStellar controllers in a WDS space (e.g., wds1) use the following parameters: 
-- `SpaceNS: wds1-system` and `KSController: kubestellar-controller-manager-metrics-service`
-- `SpaceNS: wds1-system` and `KSController: ks-transport-controller`
+To monitor the KubeStellar controllers in a WDS space (e.g., wds1) or ITS space (e.g., its1) use the following combination of `NS/APIService` parameters: 
+- `NS: wds1-system` and `KSController: kubestellar-controller-manager-metrics-service`
+- `NS: wds1-system` and `KSController: ks-transport-controller`
+
+Likewise, to monitor the KubeStellar controller in a ITS space and WEC (e.g., its1): `NS: its1-system` and `KSController: status-addon-controller` & `NS: open-cluster-management-agent-addon` and `KSController: status-agent-controller`.
+
+Note: 
```

Comment:
This turns up a lot:

```console
mspreitz@mjs13 ~ % kubectl --context kind-kubeflex get Endpoints -A | egrep "NAME|system .*controller"
NAMESPACE         NAME                                                              ENDPOINTS                                               AGE
its1-system       status-addon-controller                                           10.244.0.51:9280                                        63m
kubeflex-system   kubeflex-controller-manager-metrics-service                       10.244.0.11:8443                                        117m
wds1-system       ks-transport-controller                                           10.244.0.49:8090                                        64m
wds1-system       kubestellar-controller-manager-metrics-service                    10.244.0.48:8443                                        113m
```

Reply:
Fixed! I am using the namespace flag to restrict to the relevant namespaces only (e.g., `wds1-system` & `its1-system`).

Reply:
My point was about using Endpoints rather than a more tedious approach.

Reply:
Yes, I already updated the instructions to use the Endpoints approach. Thanks!